src/Generic.Repository/Repository/CommandAsync.cs

```diff
@@ -96,31 +96,32 @@ public virtual async Task UpdateAsync(IEnumerable<TValue> entityList, Cancellati
         #region COMMIT
 
         public async Task MultiTransactionsAsync(
-            Func<DbContext, Task> transaction,
+            Func<DbContext, Task> @action,
             CancellationToken token)
         {
             ThrowErrorIf.
-                IsNullValue(transaction, nameof(transaction), nameof(MultiTransactionsAsync));
+                IsNullValue(@action, nameof(@action), nameof(MultiTransactionsAsync));
 
-            using (var contextTransaction = Context.Database.BeginTransaction())
-            {
-                try
-                {
-                    await transaction(Context).
-                        ConfigureAwait(false);
+            var strategy = Context.Database.CreateExecutionStrategy();
 
-                    await contextTransaction.
-                        CommitAsync(token).
-                        ConfigureAwait(false);
-                }
-                catch (Exception e)
+            await strategy.ExecuteAsync(async () =>
+            {
+                using (var transaction = await Context.Database.BeginTransactionAsync())
                 {
-                    await contextTransaction.RollbackAsync(token).
-                        ConfigureAwait(false);
-
-                    throw e;
+                    try
+                    {
+                        await @action(Context);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/guilhermecaixeta/Generic.Service.DotNetCore2.2/pullRequest?prid=4960935)

Reply:
Add .ConfigureAwait();